### PR TITLE
Add Hubs link to Footer, add hover color on links

### PR DESCRIPTION
**Pull request by piperchester**
## Description

This implements #428 and adds a hover color to links. Took on a tiny refactor and consolidated the duplicate code into a `FooterLink` component.

https://user-images.githubusercontent.com/1530684/107058251-3a3a1e80-6789-11eb-9bd0-c7d2867c3304.mov

WDYT @tobiasrehm? 

## Todo

- [x] PR has a meaningful title?
- [ ] Lint passes?
- [ ] Ran `yarn format` in frontend?

### CODE REVIEW

File: frontend/src/components/footer/LargeFooter.js [hunk 1]
```diff
@@ -87,7 +91,7 @@ const useStyles = makeStyles((theme) => ({
     fontSize: 30,
     color: "black",
     "&:hover": {
-      color: "blue",
+      color: theme.palette.primary.main,
```
Comment (piperchester) [inline frontend/src/components/footer/LargeFooter.js:94]: Also added a consistent hover color that's within our theme.

**Comment by tobiasrehm** [comment]
Great addition! I like it!
You could also change the colour of the social media icons. They should actually have the main colour.

**Comment by piperchester** [comment]
Sounds good! Like this? 

https://user-images.githubusercontent.com/1530684/107126484-1b10bf00-6865-11eb-9dda-5c179a2432af.mov

**Comment by tobiasrehm** [comment]
Looking great! 
Is the frame around the newsletter section gone on purpose? 
And I would still be in favour of having a dedicated contact section

**Review by ddhanesha — APPROVED** [review]
Code looks good to me. Approved!

**Comment by piperchester** [comment]
> Is the frame around the newsletter section gone on purpose?

I believe that's just at a certain viewport size, here's a Loom: https://www.loom.com/share/88e082866fc44280aed67d1f7dee12a2

> And I would still be in favour of having a dedicated contact section

Oh I might've misunderstood you @tobiasrehm are you ok with 3 sections with 4 links each? We could rename "Legal" to "Contact" or something. If you'd rather we restore 4 sections with an odd number of links, we can do that too, will just have to refactor the HTML and CSS some.

**Review by positiveimpact — APPROVED** [review]
LGTM, too

**Comment by piperchester** [comment]
Thanks @ddhanesha and @positiveimpact - I believe @tobiasrehm would prefer having 4 columns with odd numbered links, so I'm going to update this PR to implement that instead of its current state. I'll re-tag you both for review requests.

**Comment by piperchester** [comment]
This is the final state @tobiasrehm @ddhanesha @positiveimpact 

Screen Shot 2021-02-16 at 6 40 13 AM

**Comment by piperchester** [comment]
This is the final state @tobiasrehm @ddhanesha @positiveimpact 

Screen Shot 2021-02-16 at 6 40 13 AM

**Review by positiveimpact — APPROVED** [review]
LGTM!

**Pull request merged**